src/main/java/org/ibp/api/brapi/v1/germplasm/GermplasmResourceBrapi.java

```diff
@@ -180,15 +180,7 @@
 		@PathVariable final String crop,
 		@PathVariable final String germplasmDbId) {
 
-		final int gid;
-		try {
-			gid = Integer.parseInt(germplasmDbId);
-		} catch (final NumberFormatException e) {
-			return new ResponseEntity<>(new SingleEntityResponse<Germplasm>().withMessage("no germplasm found"), HttpStatus.NOT_FOUND);
-
-		}
-
-		final GermplasmDTO germplasmDTO = this.germplasmService.getGermplasmDTObyGID(gid);
+		final GermplasmDTO germplasmDTO = this.germplasmService.getGermplasmDTObyGUID(germplasmDbId);
 
 		if (germplasmDTO != null) {
 			final ModelMapper mapper = new ModelMapper();
```

Comment:
Remove this line:  `germplasm.setCommonCropName(crop);`. 
From IBP-4336, commonCropName in the context of germplasm BrAPI calls is now an attribute